livechats/consumers.py

```diff
@@ -101,9 +102,9 @@ async def disconnect(self, close_code):
         messages = None
         async with redis.client() as conn:
```

Comment:
Consider adding error handling for database operations, especially since they're asynchronous. Ensure that appropriate error messages or responses are sent back to the client in case of failures.